### ISSUE TITLE
Improve https://ocaml.org/learn/portability.html page

### ISSUE THREAD

**Issue by nibble0101** [ocaml/v2.ocaml.org#1336]
# Issue description

When you navigate to [learn/portability](https://ocaml.org/learn/portability.html) page, you will see a couple of tables which are quite difficult to read and comprehend. You can check the screenshots below.

![image](https://user-images.githubusercontent.com/52580190/113113312-d4ab6100-9212-11eb-9553-f4ce92ec3254.png)
![image](https://user-images.githubusercontent.com/52580190/113113362-e7259a80-9212-11eb-8356-57b594b87ba9.png)

I think it would make things a little easier to understand if we add padding  to the table rows, add table outline  and alternating rows are styled to become zebra striped. I believe it will even look better. For reference you can check [this W3C example](https://www.w3schools.com/howto/howto_css_table_zebra.asp).

**Comment by nibble0101** [comment]
@patricoferris  and @avsm. What do you think?

**Comment by patricoferris** [comment]
That sounds great @nibble0101 is this something you want to work on?

**Comment by nibble0101** [comment]
@patricoferris  No. You seem overwhelmed by the number of applicants #1245 . I am just trying to help. I will work on some issues perhaps later on when the numbers reduce.

**Comment by Msgl** [comment]
Hi! my name is Maria and I am an Outreachy candidate. Can I work on this issue?

**Comment by nibble0101** [comment]
@Msgl Sure you can go ahead.

**Comment by SowmyaSv** [comment]
@avsm @patricoferris @gs0510 I would like to work on this as my good first issue.

**Comment by mercybassey** [comment]
![Screenshot from 2021-04-07 21-54-44](https://user-images.githubusercontent.com/57226464/113933152-23955f80-97ec-11eb-8d65-b0e0478db50c.png)

with this the tables are easy to read and understand. I'd love to work on this issue

**Comment by Msgl** [comment]
Hello @mercybassey ! Im still working on this issue.

**Comment by mercybassey** [comment]
x-special/nautilus-clipboard
copy
file:///home/mercybassey/Desktop/mobileview.png

and a little responsiveness on mobile view to enhance user experience @patricoferris  whats do you think

**Comment by mercybassey** [comment]
![mobileview](https://user-images.githubusercontent.com/57226464/113939777-4deb1b00-97f4-11eb-83ea-ee00b04da6b2.png)


And a little responsiveness by adding a vertical scroll bar to the table to help for easier readability @patricoferris

**Comment by gs0510** [comment]
@mercybassey  It's unfair that you went ahead and worked on this issue even though @Msgl said that they were working on this issue. There's a lot of open issues for you to work on, you can work on one of those.

**Comment by mercybassey** [comment]
I'm so sorry. I actually worked on it because I felt I  could work on it as long as it isn't assigned to anyone. Should I close the PR? @gs0510

**Comment by gs0510** [comment]
@Msgl Are you still working on this issue?

**Comment by gs0510** [comment]
> I'm so sorry. I actually worked on it because I felt I could work on it as long as it isn't assigned to anyone. Should I close the PR? @gs0510

We'll wait for @Msgl , If they are still working we can close your PR. There's a lot of open issues, you can choose from the ones that haven't been assigned.

**Comment by Msgl** [comment]
Hello @gs0510 ! Yes, I am still working on this issue.

**Comment by Lontchi12** [comment]
Hello Everybody, please i'm interested in this project and i'll like to contribute to it, please can someone show me the go ahead?

**Comment by Msgl** [comment]
Hello @Lontchi12 ! I'm still working on this issue :) 

A quick question @gs0510 can I create a PR once my solution is ready or should I wait for the other PR to be closed first?

**Comment by gs0510** [comment]
@Msgl, you can go ahead and open a PR!

**Comment by gs0510** [comment]
@mercybassey can you close your PR please? You can pick any other open issue on the repo. Thanks!
